utils/utils.c

```diff
@@ -37,6 +30,66 @@ const char *argp_program_version = ""                                 \
   "or later), or the GNU General Public License, version 2 (GPLv2),\n"\
   "in all cases as published by the Free Software Foundation.";
 
+struct gbConf *gbConf = NULL;
+
+int
+initGbConfig(void)
+{
+  pthread_mutexattr_t attr;
+  int ret;
+
+
+  if (gbConf)
+      return 0;
+
+  if (gbCtx == GB_CLI_MODE) {
+    if (GB_ALLOC_N(gbConf, sizeof(gbConf)) < 0) {
+      ret = errno;
+      MSG(stderr, "Failed to allocate memory for gbConf %s", strerror (errno));
+      return ret;
+    }
+
+    pthread_mutex_init(&gbConf->lock, NULL);
```

Comment:
@lxbsz where are we initializing the other members of gbConf  for GB_CLI_MODE ?

You have removed:

```
struct gbConf gbConf = {
	  .lock = PTHREAD_MUTEX_INITIALIZER,
	  .glfsLruCount = LRU_COUNT_DEF,
	  .logLevel = GB_LOG_INFO,
	  .logDir = GB_LOGDIR_DEF,
	  .cliTimeout = CLI_TIMEOUT_DEF
};

```

Reply:
Ah, sorry, should remove the following "return 0"